migration-to-PERCS-v2.0.md

```diff
@@ -0,0 +1,222 @@
+# Migrating from PER-CS v1.0 (PSR-12) to PER-CS v2.0 ###
+
+## Summary
+
+PER-CS is the next evolution of the PSR set of Coding Standards from the
+PHP-FIG (Framework Interoperability Group). It extends the Coding Standards
+laid out in PSR-12 to the newest functionality added to PHP such as the match
+keyword, enums, attributes and more.
+
+This document describes the changes and additions on a section by section
+basis between PER-CS v2.0 and PER-CS v1.0 which is a direct equivalent of 
+PSR12 with very minor changes.
+
+It is derived in part from [a GitHub generated changelog](https://github.com/php-fig/per-coding-style/compare/1.0.0...2.0.0)
+and focuses on the changes in a section-by-section basis as it's focus is to be more readable.
+
+The intent of this document is to provide a summary of these changes that can
+then be used to drive action-lists for toolset producers to support PER-CS v2.0.
+
+Any section not specified in this document does not have any PER-CS specific changes.
+
+## [Section 2.6](https://www.php-fig.org/per/coding-style/#26-trailing-commas)
+Numerous constructs now allow a sequence of values to have an optional trailing
+comma:
+ * If the final item is on the same line then there MUST NOT be a trailing comma
+ * If the final item is not on the same line then there MUST be a trailing comma
+
+## [Section 4.4](https://www.php-fig.org/per/coding-style/#44-methods-and-functions)
```

Comment:
The heading titles should include what the section is, not just the section reference.  As long as there's a link, the section reference isn't all that useful.  A proper title is.

Reply:
Expanded the heading titles for clarity.